frontends/p4/typeChecking/typeChecker.h

```diff
@@ -85,8 +85,8 @@ class TypeInference : public Transform {
     const IR::Node* initialNode;
 
  public:
-    // If readOnly=true it will assert that it behaves like
-    // an Inspector.
+    // @param readOnly If ftrue it will assert that it behaves like
```

Comment:
*If true

Reply:
Also, I wish there was a way to enforce this invariant better...

Reply:
This has been here for at least 7 years.
It's not too bad, but indeed, I sometimes wished there was another way to do this.

Reply:
I will fix this typo in a separate PR, so we don't have to wait for another build before merging.